merkle/sparse_merkle_tree_test.go

```diff
@@ -1,3 +1,157 @@
 package merkle
 
-import ()
+import (
+	"bytes"
+	"crypto/sha256"
+	"encoding/base64"
+	"fmt"
+	"math/big"
+	"sort"
+	"testing"
+
+	"github.com/google/trillian"
+)
+
+// This root was calculated with the C++/Python sparse merkle tree code in the
+// github.com/google/certificate-transparency repo.
+const sparseEmptyRootHashB64 = "xmifEIEqCYCXbZUz2Dh1KCFmFZVn7DUVVxbBQTr1PWo="
+
+func mustDecode(b64 string) trillian.Hash {
```

Comment:
We should probably start collecting up test helper code in one place. Can leave till later.